### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [saidaAfroj/Covid_app#3]
👍 Nice work on getting the data schema defined! Next week we jump into the first implementation sprint. You'll have 3 sprints (one week each) to complete all the required stories for your app and 2 sprints (one week each) to work on optionals and add polish. Let us know if we can help in any way!

**Comment by saidaAfroj** [comment]
Hi
I have some confusion about the sprint and this week assignment!
Can you elaborate little bit what to submit this week? Do I have to
implement some of the project? Or I have build the sprint?
Also can you tell little bit what is sprint?
Thanks

**Comment by codepathreview** [comment]
Hello there, 

The deliverables for the Unit 6 group project are the following:
- **Wireframe images**. These are the draft of how your app will look like. It involves the screens you will use and the lay-out of different components - buttons, inputs, display, etc. This is a required deliverable and is worth 4 points. 
- **Schema Section**. These are the **data models** and the **networking** that you will be using in your app. The data models are the variables with the corresponding data types. The networking are the different API commands that you might need, such as POST, DELETE and GET. The data models is worth 3 points and the networking is also worth 3 points. 
- **Digital Wireframe/Mockup Images**. This is a better look of the draft of your wireframe images. So instead of having the handwritten draft, you used some tools, such as Figma, Adobe, etc. to make the draft better. This is a bonus deliverable and is worth 1 point. 
- **Interactive Prototype gif**. This is another format of your wireframe. This might still be a draft but instead of submitting images, you submitted a GIF. This deliverable if worth 1 point. 

You can find the list of deliverables per unit in the course portal. [Here is a sample README for Unit 6 Group Project](https://github.com/codepath/me102_group_project_example#Schema). 


For Unit 7, the deliverables are as follows:
- **GitHub Project created** (1 point). You have to create a project board in your Github repository. There is a menu found on top of your repo called Projects. You can just click that and add or create a new project. 
- **GitHub Milestones created** (1point). These are the milestones of your app which also corresponds to your sprints. You already have a list of user stories, you will divide these user stories into different sprints (or weeks) during development. In the first sprint (or first week), should be about development of the base code of your app. You can create the milestones by clicking the Issues button on the menu and clicking **Milestones** (right beside the New Issue button). Then click the create milestones button. 
- **GitHub Issues created from user stories** (2 points). These are the issues that will help you work on the user stories you listed. One user story you listed is, _User can toggle between positive, negative and death cases._ You can create issues from this user story by listing down the things that you need in order to to complete this user story. The issue could be - _create a toggle button for positive cases, negative cases and death cases_. You can have multiple issues for a single user story. 
     - There are two ways to create an issue -- via the project board and via the issues tab. Once you are done creating the project and adding columns To Do, In Progress and Done. Click the + icon on the To Do column. This will allow you to create notes. Add a description of the note, it could be something like -- create a toggle button for positive cases, negative cases and death cases. You can then **convert this into an issue** by clicking the `. . .` of each note.  
     - The second way to create an issue is via the Issues menu. Click the Issues tab in the menu and click new issue. Add a title and a description. 
- **Issues added to project and assigned to specific team members** (1pt). The issues you created must be assigned to different members of the group. You can do this by opening the issue you created. In the right side, there is an option to add Assignees, Labels, Projects, and Milestone. The assignees must be the members of the group, projects will be the project that you created, and milestone will be the week in which you will work on the issue. 
- **Updated status of issues in Project board**. In the project board, you have to add notes and convert them to issue. Those issues that are yet to be work on must be in the To Do column, issues that are currently being work on must be in the In Progress column, while issues that are already done must be in the Done column. Don't forget to move one note ( or issue) from one column to another as you work on each. 
- **Sprint planned for next week (Issues created, assigned & added to project board)** (3pts). These are the different set of user stories and issues the your group have to do for the next week. There is no need to work on these issues yet, but you have to create the issues from user stories (if haven't yet), assign them to the next milestone, assign group members who will work on the issues and again, add them to the project board. You can first create the note from the project board, convert it to an issue and assign people and milestone. 
- **Completed user stories checked-off in README** (2 points). These are the user stories that you have already completed for the week. You have to mark the user stories check. We have taken a look at the user stories you provided in the assignment. These are correct but it is best to put them on the list. Something similar to your previous assignment, where the user stories a re listed, and you mark ✅ those that you completed. 
- **Gifs created to show build progress and added to README** (3pts). This is the GIF showing the user stories you completed. You are to record a new GIF for each Unit that you submit. So for Unit 7, create a GIF for unit 7. For Unit 8, you have to create a new GIF as well. Do not delete the GIF that you have for the unit 7. 

A sprint, is a period that will help you work on your app. One sprint is equivalent to one week. This means that you will work on the project, implementing user stories with the help of issues you created. Each sprint requires an output, you can show this by recording a GIF. 

In summary, the assignment that you have to submit for this week does not require you to start working on your app but you are very much welcome to do so. The progress our output of your app must be submitted by the next deadline, March 13 at 11:59 PM Pacific Time. You can refer to the course portal for the deadline. Sprint is equivalent to a week when you will work on your app. 

We hope this information helps. If you need further assistance, feel free to reach out.
